### ISSUE TITLE
New db created after update

### ISSUE THREAD

**Issue by seanthegeek** [coolacid/docker-misp#47]
I just updated to get the recent nginx config, and somehow a new db got created. What am I doing wrong?

I ran:

```bash
git pull https://github.com/coolacid/docker-misp master
docker-compose pull
docker-compose down
docker-compose up
```

**Comment by seanthegeek** [comment]
Oh...I had init set to true in docker-compose.yml. So how do I get it to use the old db container?

**Comment by seanthegeek** [comment]
It's the same container actually, but without `init: true`, the DB is empty :(

```bash
docker images

REPOSITORY             TAG                 IMAGE ID            CREATED             SIZE
coolacid/misp-docker   core-latest         3b8b1fe3c478        8 days ago          688MB
mysql                  8.0.19              9b51d9275906        2 weeks ago         547MB
coolacid/misp-docker   modules-latest      04430ec26b0a        3 weeks ago         876MB
namshi/smtp            latest              939336167e6d        5 months ago        209MB
redis                  5.0.6               de25a81a5a0b        5 months ago        98.2MB
```

**Comment by coolacid** [comment]
Nothing I see you did should create a new database, only if you `docker-compose rm` should it delete the volume. It also shows it was created 2 weeks ago. Can you exec into the mysql container and see if the data is still there? No changes were done to which DB is used. 

For "prod" I typically use an external DB anyway, and set the values in the docker-compose.

**Comment by seanthegeek** [comment]
Found and fixed the problem in PR #48

**Comment by coolacid** [comment]
My mistake here was that I always use `docker-compose up` which when you `ctrl-c` out of the container, it stops the containers, not kills them. Thus, I never had the problem, and reading the documentation says `down` doesn't delete volumes. 

I made a bad assumption that when a `dockerfile.yml` defines the volume it just worked the same. Apparently, not the case. The commit above forces docker to use an anonymous mount point for the MySQL data and lets the user delete using docker-compose way of doing things.
